packages/rhi-webgl/src/GLRenderTarget.ts

```
@@ -306,7 +307,7 @@
       const { internalFormat, attachment } =
         _depth instanceof Texture
           ? /** @ts-ignore */
-            (_depth._platformTexture as GLTexture)._formatDetail
+          (_depth._platformTexture as GLTexture)._formatDetail
```

ESLint

[error] Insert `··`

(prettier/prettier)